docs/DeveloperGuide.md

```diff
@@ -663,16 +762,26 @@ testers are expected to do more *exploratory* testing.
    1. Test case: `delete 0`<br>
       Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect `delete` commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases ... }_
-
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-</li>
+   1. Modify or delete data files
+
+   1. Open AssetBook. If AssetBook is open, close and reopen it.<br>
+      Expected: AssetBook will detect missing/corrupted data files, and start with an empty list
```

Comment:
Nit: Missing a period. Also I believe the present behavior shows a warning.

Reply:
Periods added for rest of test cases